### PR TITLE
Temporary test PR to debug fix precision of OpenCL platform on Travis

### PR DESCRIPTION
**Pull request by andrrizzi**
Temporary test PR.

### CODE REVIEW

**Comment by andrrizzi** [comment]
@jchodera Looks like [you are right](https://travis-ci.org/choderalab/yank/jobs/174690026#L1270-L1272)! There's something funny going on with the platform. I'll keep playing with this tomorrow.

**Comment by jchodera** [comment]
Feel free to post your observations on the OpenMM issue tracker if you want more eyeballs looking at the problem.

**Comment by jchodera** [comment]
This is probably low priority. We can always come back to this later.

File: Yank/tests/test_yaml.py [hunk 1]
```diff
@@ -1550,9 +1550,9 @@ def test_default_platform_precision():
             assert platform.getPropertyDefaultValue('CudaPrecision') == 'mixed'
         elif platform_name == 'OpenCL':
             if opencl_support_double:
-                assert platform.getPropertyDefaultValue('OpenCLPrecision') == 'mixed'
```
Comment (jchodera) [inline Yank/tests/test_yaml.py]: What about using a `try...except` clause so that we can support old and new OpenMM versions for testing?

Comment (andrrizzi) [inline Yank/tests/test_yaml.py]: Sure! I am just experimenting in this PR. This is not going to be merged.

**Comment by Lnaden** [comment]
Im going to close this PR as I dont think we have any use for it right now, we can reopen if its a problem.

**Pull request closed without merging**